tests/unit/recommenders/evaluation/test_python_evaluation_time_performance.py

```diff
@@ -96,7 +96,7 @@ def test_merge_rating(rating_true, rating_pred):
             col_rating=DEFAULT_RATING_COL,
             col_prediction=DEFAULT_PREDICTION_COL,
         )
-    assert t.interval < 19.58985256
```

Comment:
Still there are test failing:
```
>       assert t.interval < 19.58985256
E       assert 19.646452959000044 < 19.58985256
E        +  where 19.646452959000044 = <recommenders.utils.timer.Timer object at 0x7fa51d6e0ac0>.interval
tests/unit/recommenders/evaluation/test_python_evaluation_time_performance.py:99: AssertionError
_____________________________ test_get_top_k_items _____________________________
rating_true =          userID  itemID  rating
0          3456   15622       3
1          6070    9661       5
2          5947   1002...97    2[77](https://github.com/microsoft/recommenders/runs/7310019910?check_suite_focus=true#step:12:78)2   12248       5
9999998    9064   12630       4
9999999    4[78](https://github.com/microsoft/recommenders/runs/7310019910?check_suite_focus=true#step:12:79)7   11646       1
[10000000 rows x 3 columns]
    def test_get_top_k_items(rating_true):
        with Timer() as t:
            get_top_k_items(
                dataframe=rating_true,
                col_user=DEFAULT_USER_COL,
                col_rating=DEFAULT_RATING_COL,
                k=10,
            )
>       assert t.interval < 2.94850964
E       assert 3.1[83](https://github.com/microsoft/recommenders/runs/7310019910?check_suite_focus=true#step:12:84)069471999943 < 2.94[85](https://github.com/microsoft/recommenders/runs/7310019910?check_suite_focus=true#step:12:86)0964
E        +  where 3.18306[94](https://github.com/microsoft/recommenders/runs/7310019910?check_suite_focus=true#step:12:95)71[99](https://github.com/microsoft/recommenders/runs/7310019910?check_suite_focus=true#step:12:100)9943 = <recommenders.utils.timer.Timer object at 0x7fa514c1a220>.interval
```

There are several things we can do, we can take the MEAN + 5 * STANDARD DEVIATION, maybe take the Max + 5 STD. That or increase the mean by value by certain percent.

We should make sure that all these tests have enough room so they don't fail